api/entity/v1/reqres/policy.go

```diff
@@ -123,9 +123,15 @@ type PolicyListRequest struct {
 	PathPattern     string `json:"pathPattern"`
 }
 
+// PolicyListItem is a minimal policy representation for list operations
+type PolicyListItem struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
```

Comment:
Can we move this to `data.*`?

Reasoning: if `Policy` belonged to the data package and we are replacing it with a lighter mutation of it, then that thing should also go to the data package.